C23_WMDE_Desktop_DE_01

Sperling-0 opened the pull request:
- Archive C23_WMDE_Desktop_DE_00
- Prepare campign for C23_WMDE_Desktop_DE_01
- Implement VAR for C23_WMDE_Desktop_DE_01
    
- Ticket: https://phabricator.wikimedia.org/T340620

timEulitz commented on mobile_slider/banner_ctrl.js:
```diff
@@ -91,9 +92,11 @@ $( '.select-group__option' ).click( function () {
 $( 'input[name=interval]' ).click( function () {
 	const subPaymentButton = $( 'button[data-payment-type=SUB]' );
 	if ( $( this ).attr( 'id' ) !== 'interval_onetime' ) {
+		if ( subPaymentButton.hasClass( 'selected-option' ) ) {
```
If it's not there, `$( '#zahlweise' ).val( '' );` will always reset the input field value when Sofortüberweisung is disabled. With the check in place, it will only do that if Sofortüberweisung was previously selected. I should probably add a comment that clarifies this though.

timEulitz commented on mobile_slider/banner_ctrl.js:
```diff
@@ -118,6 +121,22 @@ $( '.payment-selection button' ).click( function ( event ) {
 } );
 
+$( '#banner-form-submit' ).click( function () {
+	if ( !$( '#periode' ).val() || jQuery( '.interval-selection .selected-option' ).length === 0 ) {
```
Nah, I goofed after checking some stuff in JavaScript console.

timEulitz commented on mobile_slider/banner_ctrl.js:
```diff
@@ -62,17 +61,7 @@ const weekdayPrepPhrase = dayName.isSpecialDayName() ? Translations[ 'day-name-p
 const $bannerContainer = $( '#WMDE-Banner-Container' );
 const CampaignName = $bannerContainer.data( 'campaign-tracking' );
 const BannerName = $bannerContainer.data( 'tracking' );
-const numberOfDaysUntilCampaignEnd = campaignDays.getNumberOfDaysUntilCampaignEnd();
```
It was only needed for the progress bar, it will be added back alongside the progress bar later during the campaign.

tzhelyazkova commented:
Tried it on Safari, it looks pretty ok to me. 
Behaves as described in the ticket, besides the clicking on everywhere on the small banner to open the form. But that I assume would be impossible for now.

timEulitz commented:
@tzhelyazkova @gbirke Please check 3d362cb and the corresponding changes I have made to the BannerEvents:

https://meta.wikimedia.org/w/index.php?title=Schema:WMDEBannerEvents&oldid=18437830

tzhelyazkova commented:
The changes seem alright to me :footprints:

tzhelyazkova commented:
Merged this because it went live and the new `EventLogging` schema was also merged and deployed to production

Sperling-0 committed:
Archive C23_WMDE_Desktop_DE_00

Ticket: https://phabricator.wikimedia.org/T340620

Sperling-0 committed:
Prepare campign for C23_WMDE_Desktop_DE_01

- The headline of the non-slider CTRL banner shows the ℹ️ icon.
- The progress bar is removed from the CTRL banner while the height of the banners is maintained.
- In the CTRL banner, the highlighted sentence is replaced by "Millionen Menschen nutzen Wikipedia, aber 99 % spenden nicht – sie übergehen diesen
Aufruf." in the CTRL non-slider and the slider version.
- In the bank account line at the bottom
	- the account holder name is changed from Wikimedia to Wikimedia e. V.
	- the bank account number is changed to DE05 1002 0500 0003 2873 00.
- The banner-closed event is triggered at a 10% rate.
- In the use of funds overlay, the font size of the annual revenue chart is increased from 14px to 16px in use of funds overlay.

Ticket: https://phabricator.wikimedia.org/T340620

Sperling-0 committed:
Implement VAR for C23_WMDE_Desktop_DE_01

- The headline of the VAR non-slider shows the ℹ️ icon.
- The progress bar is removed from the VAR banner while the height of the banners is maintained.
- Fix Campaign_info.toml

- fixed some old leftover errors in the event_map files
- disable some tests related to the address form step for the VAR banner

Ticket: https://phabricator.wikimedia.org/T340620

gbirke commented on campaign_info.toml:
```diff
@@ -6,9 +6,9 @@
-description = "3 step form vs 4 step form"
-campaign_tracking = "00-ba-230501"
+campaign = "C23_WMDE_Desktop_DE_01"
+description = "Entire banner text change"
```
```suggestion
description = "Text change: Wikipedia's usefulness"
```
Using the wording from the ticket might help later finding that ticket or finding out  what the text change was

moiikana committed:
Restart campaign with _2 naming
